src/components/Header/MobileHeader/MenuBurger/MenuBurger.tsx

```diff
@@ -16,15 +17,19 @@ import Logo from "@assets/Header/logo.svg";
 import Light from "@components/Light/Light";
 import { ThemeButton } from "@components/Buttons/ThemeButtons/ThemeButtons";
 import CloseButton from "@components/Buttons/CloseButton/CloseButton";
-import { useGetCurrentUserGroupsQuery } from "@store/Controllers/GroupsController/GroupsController";
-import { useGetCurrentUserBalanceQuery } from "@store/Controllers/UserController/UserController";
+import { ToggleCurrencyButton } from "@components/Buttons/ToggleButton/ToggleButton";
+
+
 interface IPropsMenuBurger {
     setMenuActive: (value: boolean) => void
     isMenuActive: boolean,
     User: IGetCurrentUserInfo
 }
 
-const MenuBurger: FC<IPropsMenuBurger> = ({setMenuActive, isMenuActive, User}) => {
+const MenuBurger: FC<IPropsMenuBurger> = ({ setMenuActive, isMenuActive, User }) => {
+    
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)